util/fips-checksums.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/sh
+
+HERE=`dirname $0`
+
+for f in "$@"; do
```

Comment:
Hm, so the order of the files matters.
I thought this was going to be done in two parts: sorted list of files/digests, and then the final FIPS digest is the digest of that file.

Reply:
Ah, I totally forgot about sorting.  Thank you.

Reply:
Fixed